public/sass/pages/_history.scss

```diff
@@ -0,0 +1,268 @@
+// History Table
+.history-table {
+  // .gf-form overrides
+  .gf-form-label { display: none; }
+
+  .gf-form-switch {
+    margin-bottom: 0;
+
+    input + label {
+      height: 3.6rem;
+      width: 110%;
+    }
+
+    input + label::before, input + label::after {
```

Comment:
was able to get rid of all these overrides and align the styles more with standard filter-table styles :)  (while keeping the look 95% the same) 
https://github.com/grafana/grafana/commit/88da3a99e17ef8b9d3c32d98e2c6f3eebea85ac6